### DISCUSSION TITLE
GitHub account suspended without any suspension email and the support is ignoring me.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#123376] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Yes before anyone asks me to go to the support page and raise, I'll like to tell you that i already did that.

Today my GitHub account (@krishealty) was suspended without any notice, i saw it when all of my GitHub pages were suspended, i was replying to the discussions at that time. It's been years now since i am using GitHub and my critically important repositories are in it, if my account can't be reinstated, atleast give me access to my repos for a day so that i can take a backup.

I wrote about this to the support but i don't know why they're ignoring me, can anyone of the community manager help.

account: @krishealty
Ticket ID: 2779325

**Comment by franke-biosaxs** [comment]
I am not a community manager, but I came here for a similar grievance. I opened a ticket (2770625) on a related topic a week ago and ... no reaction whatsoever. I understand the "We are experiencing high volumes and therefore, you may experience longer than normal wait times." the confirmation email warns, but is the backlog really that long? Especially as the organisation I submitted the ticket for is actually on a paid plan.

That said, if the original submitter's ticket ID is recent, and the numbering is consecutive, then GitHub received about 9000 tickets this last week. How many people actually answer these? Hmm.

In case a community manager happens to drop in, I for one would appreciate any form of feedback. Thanks.

**Reply by ghost** [reply, marked as answer]
Hi franke, thanks for replying.

We do understand the high number of tickets in a week but I've been using GitHub for 4 years now and as you said that you submitted a request for a professional account shows the weak management of customer support by a big firm like GitHub, it's very disappointing and hope we get to hear from them soon.

**Comment by LiteBrite82** [comment]
Hi @krishverse 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by Chukwuebuka-2003** [comment]
@krishealty how were you able to retrieve yours?
